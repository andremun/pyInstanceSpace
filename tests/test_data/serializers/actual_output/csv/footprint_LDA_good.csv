Row,z_1,z_2
bnd_pnt_1,0.0,0.0
bnd_pnt_2,-2.4389925676564546,-3.136702774586476
bnd_pnt_3,-2.379225457025299,-3.0500060730894383
bnd_pnt_4,-2.344944497135117,-2.154904104649589
bnd_pnt_5,-2.194516451508615,-2.098008876695699
bnd_pnt_6,-1.9810845291795336,-1.8157456955079176
bnd_pnt_7,-1.9451890775485121,-1.4587228475601892
bnd_pnt_8,-2.1241306274356164,-1.0975719588691208
bnd_pnt_9,-1.2697443479169295,-0.8522452415043379
bnd_pnt_10,-1.4258139298991865,-0.48250784258691654
bnd_pnt_11,-1.0831855968203783,-0.10686021754373295
bnd_pnt_12,-0.2446196135180835,1.1945918531420923
bnd_pnt_13,0.6325185467696893,1.578120254717662
bnd_pnt_14,1.1177619458832304,1.5237544895045954
bnd_pnt_15,1.8942751684919052,1.7169442362051512
bnd_pnt_16,1.7055984681016236,1.3304833852975932
bnd_pnt_17,1.6697422402402284,0.9411774163195287
bnd_pnt_18,1.2588743157295816,0.5365049075248682
bnd_pnt_19,0.9679411358955334,0.10486190037768084
bnd_pnt_20,1.5949845053137637,-0.3861820481670827
bnd_pnt_21,2.4539536079539266,-0.4504783690579241
bnd_pnt_22,3.0305964119064335,-0.9557307421785686
bnd_pnt_23,2.998038208038607,-1.0248838118877008
bnd_pnt_24,2.1816441008776724,-1.3300483330323989
bnd_pnt_25,1.6396177819128972,-2.163167306058883
bnd_pnt_26,0.14014812274636004,-2.4233771639144788
bnd_pnt_27,-0.5324846157499852,-2.4227490558448586
bnd_pnt_28,-0.8806456262907532,-2.9020571747019805
bnd_pnt_29,-1.9214236199539343,-3.1050223841415603
bnd_pnt_30,-2.1191654904878003,-3.722975510654927
bnd_pnt_31,-2.4389925676564546,-3.136702774586476
