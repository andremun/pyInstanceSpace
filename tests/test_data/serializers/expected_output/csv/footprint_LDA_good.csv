Row,z_1,z_2
bnd_pnt_1,-2.43899256765645,-3.13670277458648
bnd_pnt_2,-2.3792254570253,-3.05000607308944
bnd_pnt_3,-2.34494449713512,-2.15490410464959
bnd_pnt_4,-2.19451645150862,-2.0980088766957
bnd_pnt_5,-1.98108452917953,-1.81574569550792
bnd_pnt_6,-1.94518907754851,-1.45872284756019
bnd_pnt_7,-2.12413062743562,-1.09757195886912
bnd_pnt_8,-1.26974434791693,-0.852245241504338
bnd_pnt_9,-1.42581392989919,-0.482507842586917
bnd_pnt_10,-1.08318559682038,-0.106860217543733
bnd_pnt_11,-0.244619613518084,1.19459185314209
bnd_pnt_12,0.632518546769689,1.57812025471766
bnd_pnt_13,1.11776194588323,1.5237544895046
bnd_pnt_14,1.89427516849191,1.71694423620515
bnd_pnt_15,1.70559846810162,1.33048338529759
bnd_pnt_16,1.66974224024023,0.941177416319529
bnd_pnt_17,1.25887431572958,0.536504907524868
bnd_pnt_18,0.967941135895533,0.104861900377681
bnd_pnt_19,1.59498450531376,-0.386182048167083
bnd_pnt_20,2.45395360795393,-0.450478369057924
bnd_pnt_21,3.03059641190643,-0.955730742178569
bnd_pnt_22,2.99803820803861,-1.0248838118877
bnd_pnt_23,2.18164410087767,-1.3300483330324
bnd_pnt_24,1.6396177819129,-2.16316730605888
bnd_pnt_25,0.14014812274636,-2.42337716391448
bnd_pnt_26,-0.532484615749985,-2.42274905584486
bnd_pnt_27,-0.880645626290753,-2.90205717470198
bnd_pnt_28,-1.92142361995393,-3.10502238414156
bnd_pnt_29,-2.1191654904878,-3.72297551065493
